src/annotator/guest.js

```diff
@@ -151,6 +151,7 @@ export default class Guest extends Delegator {
     });
 
     this.plugins = {};
+    this.BucketBar = null;
```

Comment:
```suggestion
    /** @typedef {BucketBar|null} */
    this.bucketBar = null;
```

No reason not to use our normal public field naming conventions here. Also its helpful for readers to specify the type.

Reply:
Awesome.